includes/functions.php

```diff
@@ -2,28 +2,43 @@
 /**
  * Outputs FB IA formatted caption for an image or a post thumbnail.
  *
- * @param int $image_id Image/attachment ID. Optional.
+ * If caption string is specified - it takes precedence over getting
+ * the caption directly from image/attachment post.
  *
- * @return string|void FB IA formatted image caption.
- *                     Nothing if attachment post isn't found.
+ * @param int    $image_id Image/attachment ID. Optional.
+ * @param string $caption  Image caption string. Optional.
+ *
+ * @return string          FB IA formatted image caption.
+ *                         Empty string if attachment post isn't found.
  */
-function simple_fb_image_caption( $image_id = 0 ) {
+function simple_fb_image_caption( $image_id = 0, $caption = '' ) {
```

Comment:
I'm not a fan of this function. I think its trying to do 2 things. 

1 - output markup.
2 - maybe get caption.

Then we have to handle the conditional logic of maybe we already have a caption etc etc. Just gets confusing. 

Actually I think we should use a template file for this as thats the pattern we've established elsewhere. Then move the rendering code to render_image_markup and out of this file because its not used for anything else.  I'll whip up a PR